### ISSUE TITLE
New internal tag namespace for internal reference tags

### ISSUE THREAD

**Issue by iglocska** [MISP/MISP#1633]
The idea is to have a separate, abstracted tagging system for internal references that would be added via a separate method than regular tags (maybe on the event add/edit interface as an additional field)?

These could then be displayed separately via the index.

**Comment by mokaddem** [comment]
@iglocska Isn't it solved with the local tags? Asking because no ACL is mentioned in the issue

**Comment by iglocska** [comment]
No, this is a different story. The idea is to have a tag namespace such as "misp-internal:" that is purely used to trigger automation mechanisms. They wouldn't behave from a user's perspective like normal tags.
